pytorch-example/environment.yaml

```
@@ -0,0 +1,9 @@
+channels:
+  - conda-forge
+dependencies:
+  - coiled=0.0.40
```

This dependency should be updated to the latest coiled version, it might be better to leave it blank for now
```suggestion
  - coiled=0.0.43
```

---

I'm fine doing it that way if you prefer. My reason for pinning a version is to avoid possible breakge from dependencies. PyTorch is pinned because this is a demo and presumably won't be updated, and I don't know if there are any shared dependencies between coiled and PyTorch that might cause trouble down the line if one of them is the latest version and the other is pinned.

If we're not going to specify a coiled version, I guess dask shouldn't have one either?

---

Dask should be fine. The reason why we need coiled on the latest is because you can’t use older versions of coiled, you will get a error message telling you to update coiled

---

That makes sense. I've fixed it now. And I'm glad you liked the notebook. Thanks! :)

---

Thank you, both!